schemas/sublime-build.json

```
@@ -114,9 +114,9 @@
       "type": "boolean"
     },
     "syntax": {
-      "markdownDescription": "A string specifying the syntax file to use to highlight the build system output panel.\nExample: `\"Packages/JavaScript/JSON.sublime-syntax\"`",
+      "markdownDescription": "A string specifying the syntax file to use to highlight the build system output panel.\nExample: `\"Packages/JavaScript/JSON.sublime-syntax\"` or `\"scope:source.json\"`",
       "type": "string",
-      "pattern": "^(([^/]+|Packages/.+)\\.sublime-syntax|Packages/.+\\.(hidden-)?tmLanguage)$"
+      "pattern": "^(([^/]+|Packages/.+)\\.sublime-syntax|Packages/.+\\.(hidden-)?tmLanguage)$|^scope:(?=\\w)"
```

Why using positive lookahead? It doesn't really matter if it's consumed or not.
Also, maybe put it inside the main group and change the match after `scope:` to more strict `[\w.]+`?

```suggestion
      "pattern": "^(([^/]+|Packages/.+)\\.sublime-syntax|Packages/.+\\.(hidden-)?tmLanguage|scope:[\\w.]+)$"
```

---

i have a dash in the scope of my actual use case. a underscore is also a valid one. not sure about other chars.

---

Should we then just figure out what exactly is allowed and only allow that? Otherwise we could just do a loose match like `scope:\w.+` if we don't really want to be very strict.

---

`\w` matches `_` but not `-`. I guess `[\\w.\\-]+` is enough.